charts/tidb-backup/values.yaml

```diff
@@ -72,6 +83,7 @@ tikvGCLifeTime: 720h
 # backup to or restore from gcp bucket, the backup path is in the form of <clusterName>-<name>
 gcp: {}
   # bucket: ""
+  # secretName is not necessary on GKE if you use the workload identity feature
```

Comment:
No need to inject any credentials to the job container?

Reply:
that's correct. If you enable the workload identity feature you just set a service account.